src/main/resources/Message_ja.properties

```diff
@@ -0,0 +1,33 @@
+welcome = ○×ゲーム. ようこそ!
+
+back = 0. バック
+
+menu = オプションを選択:
+playChoice = 1. プレー
+languageChoice = 2. 言語を変えてください
+exitChoice = 0. エグジット
+
+wrongMenuInput = 有効な番号ではありません!
+
+changeLanguageMenu = 言語を選択してください:
+english = 1. English
+polish = 2. Polski
+japanese = 3. 日本語
+
+firstPlayerName = 最初のプレーヤーの名前を入力します:
+secondPlayerName = 2番目のプレーヤーの名前を入力します:
+
+signOfFirstPlayer = どのサインを選びますか? [O 又は X]:
+
+boardDimensions = Type the board dimension (3 or greater):
+wrongDimensions = Number should be greater or equals 3!
+
+winningRuleQuestion = How many fields are in winning rule? [3 - n (board dimension)]:
+wrongRule = Rule should be greater or equals 3 and equals or lesser than board dimensions!
+
+startingPlayerQuestion = Which player should start the game? [1 - 2]:
+
+fieldId = Type id of selected field:
+notInRange = Id is not in range!
```

Comment:
why it is not a japanese? xd

Reply:
Due to not providing language selection at the moment, I will fix that later.